docker-gc

```diff
@@ -241,11 +242,23 @@ xargs -n 1 $DOCKER inspect -f '{{.Image}}' 2>/dev/null |
 sort | uniq > images.used
 
 # List images to reap; images that existed last run and are not in use.
-$DOCKER images -q --no-trunc | sort | uniq > images.all
+echo -n "" > images.all
+$DOCKER images | while read line
+do
+    awk '{print $1};'
+done | sort | uniq | while read line
+do
+  $DOCKER images --no-trunc --format "{{.ID}} {{.CreatedAt}}" $line \
+    | sort --key 2 --reverse \
```

Comment:
[You are right](https://www.gnu.org/software/coreutils/manual/html_node/sort-invocation.html):

> ‘--key=pos1[,pos2]’
> Specify a sort field that consists of the part of the line between pos1 and pos2 (or the end of the line, if pos2 is omitted), inclusive.

I was assuming that `sort --key 2` mean sort by field 2 _only_, I didn't realize that field 2 until the end of the line would be used. Thanks for the clarification